examples/postgres-RDS/rds-restore-blueprint.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: rds-postgres-bp
+actions:
+  backup:
+    outputArtifacts:
+      cloudObject:
+        keyValue:
+          backupLocation: "{{ .Phases.pgDump.Output.backupLocation }}"
+    phases:
+    - func: KubeTask
+      name: pgDump
+      objects:
+        pgSecret:
+          kind: Secret
+          name: '{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}-postgresql'
+          namespace: '{{ .StatefulSet.Namespace }}'
+      args:
+        image: ghcr.io/kanisterio/postgres-kanister-tools:0.84.0
+        namespace: '{{ .StatefulSet.Namespace }}'
+        command:
+        - bash
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          export PGHOST='{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}-postgresql.{{ .StatefulSet.Namespace }}.svc.cluster.local'
+          export PGUSER='postgres'
```

Comment:
Can this be made dynamic? Or does the helm chart 100% forces the user to be named only postgres? Otherwise, can we fetch the username from the Secret just like the password? Or from some other place? ConfigMap?

Reply:
I think this is the default username that gets sets for postgresDB